src/apps/frontend/services/task.service.ts

```diff
@@ -0,0 +1,87 @@
+import APIService from './api.service';
+
+export interface TaskPayload {
+    taskId: string,
+    title: string;
+    description: string;
+    type: 'Official' | 'Personal' | 'Hobby';
+    dueDate: string;
+}
+
+export interface ApiResponse<T> {
+    data: T;
+    message: string;
+    success: boolean;
+}
```

Comment:
can use the same API response that already exist in boilerplate so that it is alogned with project structure